analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -144,14 +135,16 @@ class GoMod(
     }
 
     private fun getVendorModules(projectDir: File): Set<Identifier> {
-        val vendorModulesJson = run("list", "-json", "-m", "-mod=vendor", "all", workingDir = projectDir)
+        return run(projectDir, "mod", "vendor", "-v")
```

Comment:
I'd say either keep the variable for the `std out / err` or strip the `curly braces` from the function body using `=` instead.

Reply:
I have made it an expression function.